### PR TITLE
Create 2024-04-18.mdx

### PR DESCRIPTION
**Pull request by Danasacco**

### CODE REVIEW

File: meeting-notes/2024-04-18.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+title: '2024-04-18'
```
Comment (ElliotFriend) [inline meeting-notes/2024-04-18.mdx:1]:
```suggestion
---
title: '2024-04-18'
```

Comment (ElliotFriend) [reply meeting-notes/2024-04-18.mdx:1]: You'll need those three dashes at the top of the "frontmatter," too. Otherwise, it doesn't get parsed right.

**Review by ElliotFriend — CHANGES_REQUESTED** [review]
looks good, except for a small markdown syntax issue.

**Comment by ElliotFriend** [comment]
Also, it looks like that google drive file isn't shared right: https://developers-pr530.previews.kube001.services.stellar-ops.com/meetings/2024/04/18

Screenshot 2024-04-30 at 9 11 27 AM

**Review by namankumar — COMMENTED** [review]
lets gooo

**Review by namankumar — APPROVED** [review]

**Review by namankumar — APPROVED** [review]

**Review by ElliotFriend — APPROVED** [review]
awesome! Thank you!

**Pull request merged**